notification-alert.tf

```diff
@@ -0,0 +1,20 @@
+module "reform-scan-notifications-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Reform_Scan_notification"
+  alert_desc = "Triggers when notification service receives at least 5 notifications within a 30 minutes window timeframe."
+
+  app_insights_query = "traces | where message startswith 'Started processing notification message'"
+
+  frequency_in_minutes       = 15
```

Comment:
would it cause double alerts?

Reply:
I do not think it is a big issue. A bigger issue is a missing notification